packages/data-sdk/src/AppRtcClientPool.ts

```diff
@@ -0,0 +1,77 @@
+import { RtcClient, SignalingPromiseClient } from "@formant/realtime-sdk";
+import { RtcClientPool } from "./utils/RtcClientPool";
+import { FORMANT_API_URL } from "./config";
+import { Authentication } from "./Authentication";
+import { defined } from "../../common/defined";
+
+export type { PooledRtcClient } from "./utils/RtcClientPool";
+
+const getToken = async () =>
+  defined(Authentication.token, "Realtime when user isn't authorized");
+
+const NamedRtcClientPools = {
+  unknown: new RtcClientPool({
+    ttl: 2_500,
+    createClient: (receiveFn) =>
+      new RtcClient({
+        signalingClient: new SignalingPromiseClient(FORMANT_API_URL),
+        getToken,
+        sessionType: 0,
```

Comment:
yea, I _want/would-love_ to do that, but I think i need to change/update the way that `RtcClient` gets exported and republish it from the `telemetry` side, otherwise I'd have to dig into the _package_ source itself (because its not presently exported from the top-level)

Reply:
Totally fair. Something to think about for the future.

Reply:
yea, i think if this was exported in a cleaner way it would be easier to get at. I just tried digging down and grabbing it from:

```
import { SessionType } from "@formant/realtime-sdk/dist/protos/api/signaling/v1/signaling_pb";
```

but then the build started to try to package up some of the protobuf stuff that… is _probably_ less than ideal:

```
vite v4.3.3 building for production...
transforming (162) node_modules/rollup-plugin-node-polyfills/polyfills/zlib-lib/zstream.js"google-protobuf/google/protobuf/wrappers_pb.js" is imported by "google-protobuf/google/protobuf/wrappers_pb.js?commonjs-external", but could not be resolved – treating it as an external dependency.
"google-protobuf" is imported by "google-protobuf?commonjs-external", but could not be resolved – treating it as an external dependency.
Use of eval in "node_modules/@formant/realtime-sdk/dist/index.js" is strongly discouraged as it poses security risks and may cause issues with minification.
Use of eval in "node_modules/@formant/realtime-sdk/dist/index.js" is strongly discouraged as it poses security risks and may cause issues with minification.
Use of eval in "node_modules/@formant/realtime-sdk/dist/index.js" is strongly discouraged as it poses security risks and may cause issues with minification.
✓ 183 modules transformed.
dist/data-sdk.es.js  1,337.78 kB │ gzip: 268.19 kB
No name was provided for external module "date-fns" in "output.globals" – guessing "dateFns".
No name was provided for external module "google-protobuf" in "output.globals" – guessing "require$$0".
No name was provided for external module "google-protobuf/google/protobuf/wrappers_pb.js" in "output.globals" – guessing "require$$1".
```

so, instead, _for now_ I replicated the enum here, rather than using magic numbers, which seems _"better"_. But yea, still less-than-ideal.